Tests/MongoSwiftTests/BSONValueTests.swift

```diff
@@ -135,13 +135,13 @@ final class BSONValueTests: MongoSwiftTestCase {
             return
         }
 
-        expect(_id.oid).to(equal(objectId.oid))
+        expect(_id.hex).to(equal(objectId.hex))
```

Comment:
have we lost equality of the discrete type? Can we still do: `expect(_id).to(equal(objectId))`, and if so should a test be added?

Reply:
Nope, it just wasn't tested here probably because the underlying `Equatable` conformance was just comparing the hex strings. Since I've updated the conformance to something else, I added a test here and below.

Reply:
Great, thanks